releasenotes/notes/jmxfetch-0.49.7-578d16bad0959d71.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Upgraded JMXFetch to `0.49.7 <https://github.com/DataDog/jmxfetch/releases/0.49.7>`_ which switches from snakeyaml to snakeyaml-engine,
+    adding support for YAML 1.2.
+    See `0.49.7  <https://github.com/DataDog/jmxfetch/releases/tag/0.49.7>`_ for more details.
```

Comment:
```suggestion
    Upgraded JMXFetch to `0.49.7 <https://github.com/DataDog/jmxfetch/releases/0.49.7>` which switches from snakeyaml to snakeyaml-engine,
    adding support for YAML 1.2.
    See `0.49.7  <https://github.com/DataDog/jmxfetch/releases/tag/0.49.7>` for more details.
```

Reply:
I'm thinking that the underscores might have been added here by mistake, because I'm not sure what purpose they serve in this context. Feel free to ignore this suggestion if they're needed for any reason.